pylint/pyreverse/vcg_printer.py

```diff
@@ -154,20 +154,29 @@
     NodeType.CLASS: "box",
     NodeType.INTERFACE: "ellipse",
 }
+# pylint: disable-next=consider-using-namedtuple-or-dataclass
```

Comment:
Should we use namedtuple or dataclass @DudeNr33 ? 😄 It seems like it, but I don't know how hard it would be to do that in pyreverse. Should we create another issue to fix that later ?

Reply:
I will create a separate issue for this. 👍